sormas-ui/src/main/java/de/symeda/sormas/ui/symptoms/SymptomsForm.java

```diff
@@ -994,6 +1002,43 @@ private void addListenerForOnsetFields(ComboBox onsetSymptom, DateField onsetDat
 		onsetDateField.setEnabled(false); // will be updated by listener if needed
 	}
 
+	// event for the symptomatic field
+	private void addListenerForSymptomatic(ComboBox onsetSymptom, DateField onsetDateField) {
+		Field<YesNoUnknown> sourceFieldSymptomatic = (Field<YesNoUnknown>) getFieldGroup().getField(SYMPTOMATIC);
+		List<String> allPropertyIds =
+			Stream.concat(unconditionalSymptomFieldIds.stream(), conditionalBleedingSymptomFieldIds.stream()).collect(Collectors.toList());
+		allPropertyIds.add(LESIONS_THAT_ITCH);
+
+		sourceFieldSymptomatic.addValueChangeListener(event -> {
+			for (Object sourcePropertyId : allPropertyIds) {
+				Field<SymptomState> symptom = (Field<SymptomState>) getFieldGroup().getField(sourcePropertyId);
+				if (sourceFieldSymptomatic.getValue() != null) {
+					if (!sourceFieldSymptomatic.getValue().equals(YesNoUnknown.NO)) {
+						symptom.setEnabled(true);
+					} else {
+						symptom.setValue(SymptomState.NO);
+						symptom.setEnabled(false);
+					}
+				} else {
+					symptom.setEnabled(true);
+				}
+			}
+			if (sourceFieldSymptomatic.getValue() != null) {
+				if (sourceFieldSymptomatic.getValue().equals(YesNoUnknown.NO) || sourceFieldSymptomatic.getValue().equals(YesNoUnknown.UNKNOWN)) {
+					onsetDateField.setVisible(false);
```

Comment:
I'd strongly recommend to use setEnabled(false) instead of setVisible(false) because otherwise users will be confused why the fields are suddenly missing